<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWidget</class>
 <widget class="QWidget" name="MyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyWidget</string>
  </property>
  <widget class="QPushButton" name="openSerialPort">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>打开串口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="model">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>70</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="run">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>710</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>运动</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeSerialPort">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭串口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="init">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>710</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>初始化</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>调试角度(单位/qc)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="debug">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>110</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>调试</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>62</y>
     <width>276</width>
     <height>82</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="flap_motor1">
        <property name="text">
         <string>上部电机</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="com1"/>
      </item>
      <item>
       <widget class="QLineEdit" name="debug_angle1">
        <property name="inputMask">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="pitch_motor1">
        <property name="text">
         <string>俯仰电机</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="com2"/>
      </item>
      <item>
       <widget class="QLineEdit" name="debug_angle2">
        <property name="inputMask">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="attack_motor1">
        <property name="text">
         <string>倾转电机</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="com3"/>
      </item>
      <item>
       <widget class="QLineEdit" name="debug_angle3">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
